design-system/src/_js/includes/tabs.js

```diff
@@ -0,0 +1,351 @@
+const $ = require('jquery');
+const Handlebars = require('handlebars');
+
+/* Accessible tabs > accordian */
+/* replace this JS handlebars implementation */
+/* eslint-disable */
+(function () {
+  var Tabs;
+
+  Tabs = (function () {
+
+    var templates = {
+      tplTabNav: Handlebars.compile("<ul class='coop-list-inline ds-tabs' role='tablist'>{{#each tab}}<li role='presentation' class='ds-tabs__item'><a href='#{{tabId}}' id='TabController-{{tabId}}' class='btn-pill' role='tab' aria-selected='false' aria-controls='{{tabId}}' tabindex=-1 aria-expanded='false' data-title='{{tabTitle}}'>{{tabTitle}}</a></li>{{/each}}</ul>")
+    };
+
+    /**
+     * Sets up the Tabs
+     *
+     * @param $container - parent of the items that will be tabbed together
+     * @param $options - any overrides to the classes set below
+    */
+    function Tabs($container, options) {
+
+      var defaults = {
+        default_tab: "0",                          // index of tab to open on page load
+        tab_class_panel: ".tabs-container__panel", // wrapper for each tab/accordion title and content
+        tab_class_title: ".tabs-container__title", // title element for each tab/accordion
+        tab_nav_id: "TabNav"                       // ID to provide the constructed tab navigation
+      };
+
+      this.$container = $container.addClass("tabs-init");
+      this.options = $.extend({}, defaults, options);
+      this.currentTab = null;
+      this.init();
+
+    }
+
+    /**
+     * Creates a data object for all tabs within the widget
+     * Saves each tab ID and title, to be used to create desktop tab nav if needed
+     * Attaches Aria roles as it fetches tab data
+    */
+    Tabs.prototype.fetchTabData = function () {
+      // stores data for all tabs in the widget
+      this.tabData = [];
+
+      var i = 0,
```

Comment:
Not a massive one but think there are some files that haven't had the linter run against them whilst some have.
Noticed you'd taken out the ignore design system flag so thought it worth mentioning! :)

Reply:
Good spot! Yeah I’ve wrapped all the legacy Design System files with…

```js
/* eslint-disable */
...
/* eslint-enable */
```

Wanted to keep the scope of this PR to “Add form validation demo” without rewriting old stuff where possible.